pkg/api/api_test.go

```diff
@@ -44,7 +48,7 @@ func TestNewAPI(t *testing.T) {
 			err: multierror.NewPrefixed("invalid api config",
 				errors.New("client cannot be empty"),
 				errEmptyAuthWriter,
-				errors.New("host cannot be empty"),
+				errESSInvalidAuth,
```

Comment:
I think it would make sense to write out the error here. In case a letter gets erased from the var by mistake or whatever 🤔

Reply:
fixed!